### PR TITLE
feat: add polygon poi contract

### PR DESCRIPTION
**Pull request by agusaldasoro**
Add contract for POIs in Polygon

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,3816 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (menduz) [inline yarn.lock:2]: This project uses npm and package-lock.json. would you either delete this file or migrate the whole scripts to yarn?

**Review by menduz — CHANGES_REQUESTED** [review]

File: src/ListAbi.ts [hunk 2]
```diff
@@ -2,7 +2,8 @@ import { AbiItem, BigNumber } from 'eth-connect'
 
 export const poiDeployments = {
   '5' /* goerli  */: '0xb8c7a7afd42675ab61f0a3732f8d0491825a933b',
```
Comment (marianogoldman) [inline src/ListAbi.ts:4]: Should we remove goerli and add an entry for polygon mumbai (the testnet)?

File: src/ListAbi.ts [hunk 3]
```diff
@@ -2,7 +2,8 @@ import { AbiItem, BigNumber } from 'eth-connect'
...
-  '1' /* mainnet */: '0x0ef15a1c7a49429a36cb46d4da8c53119242b54e'
+  '1' /* mainnet */: '0x0ef15a1c7a49429a36cb46d4da8c53119242b54e',
+  '137' /* polygon */: '0xFEC09d5C192aaf7Ec7E2C89Cc8D3224138391B2E'
```
Comment (marianogoldman) [inline src/ListAbi.ts:6]: Also, not sure if this is important or not, but previous entries were all lowercase...

**Review by menduz — APPROVED** [review]

File: src/ListAbi.ts [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import { AbiItem, BigNumber } from 'eth-connect'
 
 export const poiDeployments = {
-  '5' /* goerli  */: '0xb8c7a7afd42675ab61f0a3732f8d0491825a933b',
-  '1' /* mainnet */: '0x0ef15a1c7a49429a36cb46d4da8c53119242b54e'
+  '80001' /* mumbai  */: '0x08E5a5288D6bBa9383724C57175C03A37fe83A2A',
+  '137' /* polygon */: '0xFEC09d5C192aaf7Ec7E2C89Cc8D3224138391B2E'
```
Comment (marianogoldman) [inline src/ListAbi.ts:5]: Where do these contract ids come from? Just checked [here](https://contracts.decentraland.org/links.html) and they are not mentioned. Should they be added there too, for reference?

**Review by marianogoldman — APPROVED** [review]

File: src/ListAbi.ts (hunk 4 above)
Comment (agusaldasoro) [reply src/ListAbi.ts:5]: done!

**Pull request merged**